devtools/client/netmonitor.properties

```diff
@@ -1137,6 +1137,12 @@ netmonitor.context.copyAllAsHar = Copy All As HAR
 # LOCALIZATION NOTE (netmonitor.context.copyAllAsHar.accesskey): This is the access key
 # for the Copy All As HAR menu item displayed in the context menu for a network panel
 netmonitor.context.copyAllAsHar.accesskey = o
+# LOCALIZATION NOTE (netmonitor.context.saveAsHar): This is the label displayed
+# on the context menu that saves the selected request as HAR format
+netmonitor.context.saveAsHar = Save As HAR
+# LOCALIZATION NOTE (netmonitor.context.saveAsHar.accesskey): This is the access key
+# for the Save As HAR menu item displayed in the context menu for a request
+netmonitor.context.saveAsHar.accesskey = Z
```

Comment:
This is likely going to change.
https://phabricator.services.mozilla.com/D232792#inline-1297629